utils/constants.ts

```diff
@@ -13,6 +13,7 @@ export const STANDARD_NATURAL_UNIT = ether(1);
 export const STANDARD_COMPONENT_UNIT = ether(1);
 export const UNLIMITED_ALLOWANCE_IN_BASE_UNITS = new BigNumber(2).pow(256).minus(1);
 export const MAX_DIGITS_IN_UNSIGNED_256_INT = 78;
+export const ZRX_TOKEN_ADDRESS = '0x871dd7c2b4b25e1aa18728e9d5f2af4c4e431f5c';
```

Comment:
Are we storing the other testRPC (e.g. 0x V2 Exchange, etc.) addresses anywhere else or using?

Reply:
Or why don't we put it into TestUtils like the other addresses?

Reply:
Next PR, it's not currently being used as part of the tests, I just wanted to test making sure the issuance order filler had some ZRX and that we could access the ZRX token contract on the snapshot correctly.